documentation/Sand.md

```diff
@@ -26,5 +26,5 @@ But as time passed it became clear that 1) was not that strong a reason. ERC-777
 
 Secondly, we found out we could design meta-transaction outside of the SAND contract, while preserving efficiency.
 
-The new contract can be found here : [Sand.sol](old_src/Sand.sol)
+The new contract can be found here : [SandBaseToken.sol](../src/solc_0.8/Sand/SandBaseToken.sol)
```

Comment:
Actually it's not this file but the one in 0.5 that is used on mainnet

Reply:
I changed the link destination to 0.5.